.travis/after_success.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -e
+set -ev
```

Comment:
Use -x, that expands the variables before printing. Also, this could be part of the shebang line.